packages/default/scss/slider/_layout.scss

```diff
@@ -127,6 +127,11 @@
         .k-slider-bottomright .k-last { background-position: -152px 0; }
         .k-slider-bottomright .k-tick-large.k-last { background-position: -62px 0; }
 
+        .k-middle {
```

Comment:
Why `k-middle`? A belive `k-center` is better for a class that assigns `align-items: center;`

Reply:
It is related to [this issue](https://github.com/telerik/kendo-angular-inputs/issues/436) where we were about to render new, `k-middle` class.

However, it turned out that we do not need a new class, we can use the existing `k-tick-large` instead (which I did).

Reply:
If we no longer need this class, I believe you should remove/modify the test.